vlttng/venv.py

```diff
@@ -554,7 +554,11 @@ def _build_lttng_tools(self):
         if 'lttng-ust' in self._profile.projects:
             add_args = '--with-lttng-ust-prefix={}'.format(shlex.quote(self._paths.usr))
         else:
+            # LTTng-Tools prior to v2.8 was using a different flag to turn off
+            # UST support. We add both to the configure script arguments since the
+            # unsupported one will be ignored.
             add_args = '--without-lttng-ust'
+            add_args += ' --disable-lttng-ust'
```

Comment:
Why not:

    add_args = '--without-lttng-ust --disable-lttng-ust'

Also, the `--disable-lttng-ust` option (and `--enable-lttng-ust=no` I guess) should be detected in line 550 above for the warning message. At this point I would add all those strings to a list and for each of them check if its in `project.configure` (and generate that part of the warning message with `'/'.join(the_list)`).